recipes/cadd-scripts/cadd-install-wrapper.sh

```diff
@@ -0,0 +1,5 @@
+# Wrapper for CADD.sh
+set -eu -o pipefail
+
+CADD=$PACKAGE_HOME \
+    $PACKAGE_HOME/CADD.sh "$@"
```

Comment:
should this be cadd-install? Otherweise the cadd-installwrapper looks the same as the cadd-wrapper

Reply:
or `install.sh`?

Reply:
`install.sh`